drivers/docker/config.go

```diff
@@ -278,7 +278,7 @@ var (
 				"labels":  hclspec.NewBlockAttrs("labels", "string", false),
 				"driver_config": hclspec.NewBlockSet("driver_config", hclspec.NewObject(map[string]*hclspec.Spec{
```

Comment:
I think there is another bug in this line, where it should be - I'll follow up in another PR.

```suggestion
				"driver_config": hclspec.NewBlock("driver_config", false, hclspec.NewObject(map[string]*hclspec.Spec{
```